### DISCUSSION TITLE
OAuth 2.0 | Option to "Authorize using browser" in browser, not Insomnia window

### DISCUSSION THREAD

**Discussion by vibraniumforge** [Kong/insomnia#5838] [Ideas] [answered]
Hello,

It seems that a recent update has changed the functionality between Windows and MacOs versions with regard to the OAuth 2.0 popup window. On MacOs, using OAuth 2.0 it seems to open a browser window to do this. Whereas in Windows, it opens an Insomnia window to do this. The problem for me is that our organization does not allow this kind of window. It only allows Firefox, Edge, and Chrome. I get an error saying something like "Please use Edge, Chrome or Firefox 91+..." Can this functionality be reverted to what it was a few months ago? Thanks a lot!

**Comment by yaoweiprc** [comment, marked as answer]
Will be implemented in this PR https://github.com/Kong/insomnia/pull/8863
